setup.py

```diff
@@ -41,6 +41,8 @@
 pandas_requires = ['multipledispatch']
 clickhouse_requires = ['clickhouse-driver']
 bigquery_requires = ['google-cloud-bigquery']
+csv_requires = pandas_requires + ['tables>=3.0.0']
```

Comment:
This looks like it should be `hdf_requires` instead.

Reply:
done